lib/pngstream.js

```diff
@@ -30,32 +31,37 @@ var Stream = require('stream').Stream;
  */
 
 var PNGStream = module.exports = function PNGStream(canvas, sync) {
```

Comment:
I would prefer throwing an error since this is the default behaviour with ES6 classes which we probably will want to upgrade the code base to at some point. Also, you can always stop throwing an exception without a major bump but going the other way is more complicated.

This is the exact error the Node throws:

```javascript
throw new TypeError("Class constructors cannot be invoked without 'new'")
```